build.gradle

```diff
@@ -10,13 +10,16 @@ repositories {
 }
 
 dependencies {
-    testImplementation 'org.testng:testng:7.6.1'
-    implementation 'org.seleniumhq.selenium:selenium-java:4.5.0'
+    testImplementation 'org.testng:testng:7.7.0'
+    implementation /org.seleniumhq.selenium selenium.minus(java) 4.5.0/
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it.

![image](https://github.com/Test-Pro-Bootcamp/A59-QA-Automation-Framework/assets/104761482/b7aa6283-81de-4224-8bbe-a7f5ec91db27)